sources/packages/api/src/services/designation-agreement/designation-agreement-locations.service.ts

```diff
@@ -43,4 +44,59 @@ export class DesignationAgreementLocationService extends RecordDataModelService<
         "designationAgreementLocation.institutionLocation = location.id",
       );
   }
+
+  /**
+   * Service to get designation agreement locations and their designation statuses.
+   * @param institutionId
+   * @returns designation agreement locations and their designation statuses.
+   */
+  getAllDesignatedAgreementLocations(
```

Comment:
Does this method return all the locations when there absolutely no designation agreements created?